pipeline/manual_e2e_test.py

```diff
@@ -53,21 +53,21 @@
 #
 # These files contain real sample data, usually 4 measurements each, the first 2
 # are measurements that succeeded, the last two are measurements that failed.
-def local_data_to_load_1(*_) -> List[str]:
+def local_data_to_load_1(*_) -> List[str]:  # type: ignore
   return [
       'pipeline/e2e_test_data/http_results.json',
       'pipeline/e2e_test_data/https_results.json'
   ]
 
 
-def local_data_to_load_2(*_) -> List[str]:
+def local_data_to_load_2(*_) -> List[str]:  # type: ignore
```

Comment:
Why are we ignoring the type here?

Reply:
Mypy wants me to specify types for the `*_` arg, which I explicitly not defining since it doesn't matter.

I've removed the ignored and added the List[Any] type which also works.